backend/api/Controllers/Models/ScheduleLocalizationMissionQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Api.Controllers.Models
+{
+    public struct ScheduleLocalizationMissionQuery
+    {
+        public string RobotId { get; set; }
+        public string DeckId { get; set; }
+    }
+}
```

Comment:
Can remove this file and then in the RobotController:
public async Task<ActionResult<Mission>> StartLocalizationMission(
        [FromRoute] string robotId, [FromRoute] string deckId
    )